### PR TITLE
#58 Implement Strimzi

### LINKED ISSUE #58: Implement Strimzi

### PR DESCRIPTION
**Pull request by ilisau**
In this PR I implemented Strimzi and close #58

---

## PR-Codex overview
This PR updates the Kafka setup for the project, changing the host name and adding a persistent volume configuration.

### Detailed summary
- Deleted Kafka and Zookeeper YAML files
- Updated `KAFKA_HOST` in `qaprotours-configmap.yml`
- Added `kafka-clusterrolebinding.yml` and `qaprotours-service-account.yml`
- Added `kafka-setup.sh` script to install and configure Kafka
- Added `my-cluster` Kafka configuration to `kafka-persistent.yml`
- Updated `run.sh` to remove Kafka and Zookeeper service and deployment YAML files, and to apply the new Kafka configuration.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: kafka-setup.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+kubectl create namespace kafka
+kubectl apply -f https://strimzi.io/install/latest?namespace=kafka -n kafka
```
Comment (h1alexbel) [inline kafka-setup.sh:2]: why is the latest one?

prefer fixed versions

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: kafka-setup.sh (hunk 1 above)
Comment (ilisau) [reply kafka-setup.sh:2]: they have only latest tag accessible by fetching from url

Comment (h1alexbel) [reply kafka-setup.sh:2]: fix it, we need to use only prefixed versions

File: kafka-setup.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+kubectl create namespace kafka
+kubectl apply -f https://strimzi.io/install/0.34.0?namespace=kafka -n kafka
+kubectl apply -f https://strimzi.io/examples/0.34.0/kafka/kafka-persistent-single.yaml -n kafka
```
Comment (h1alexbel) [inline kafka-setup.sh:3]: @ilisau I got this error after applying it:
`unable to read URL "https://strimzi.io/install/0.34.0?namespace=kafka", server reported 404 Not Found, status code=404`

**Review by h1alexbel — CHANGES_REQUESTED** [review]

**Review by h1alexbel — APPROVED** [review]

**Pull request merged**